infra/machines/vcsa/create-esxi-vm.sh

```diff
@@ -109,22 +110,22 @@ if [ "$(govc env -x GOVC_URL_HOST)" = "." ] ; then
 
     cat > "$vmx" <<EOF
 config.version = "8"
-virtualHW.version = "11"
+virtualHW.version = "13"
```

Comment:
These changes work ok with Fusion 8.x?  I'm still on 7.x and the current set of properties works with esx 6.5.. but suppose I should upgrade at somepoint.